### ISSUE TITLE
problem with 0.4.7 release

### ISSUE THREAD

**Issue by demianh** [ngsru/vue-server#9]
I just upgraded from 0.4.6 to 0.4.7 and now my scripts are failing.

My test script only contains one line:

`var vueServer = require('vue-server');`

This is the error i get: 

```
vagrant@host:/vagrant/test$ node vue-server-test.js 

/vagrant/node_modules/vue-server/node_modules/vue/src/config.js:1
(function (exports, require, module, __filename, __dirname) { import { compile
                                                              ^^^^^^
SyntaxError: Unexpected reserved word
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/vagrant/node_modules/vue-server/src/compiler/index.js:27:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

I installed the library using `npm install vue-server`.

I noticed that the "vue" dependency (which causes the error) was not installed in 0.4.6

Edit: I guess this error is due to my node version. I am using v0.10.37 from the ubuntu repository.

Edit2: Also, the tests are failing with the same error: https://travis-ci.org/ngsru/vue-server

**Comment by demianh** [comment]
Fixed with the 0.4.8 release. Thanks!

**Comment by fullfs** [comment]
@demianh sorry didn't respond you in time :) Thanks for reporting )
